infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/InfinispanRemoteProperties.java

```diff
@@ -12,6 +12,13 @@
 /**
  * Properties for configuring the Infinispan Remote datastore via {@code persistence.xml} or
  * {@link StandardServiceRegistryBuilder}.
+ * <p>
+ * It is possible to configure the Hot Rod (Infinispan remote) client prefixing them with 'hibernate.ogm.infinispan_remote.client.'.
+ * For example, the property 'infinispan.client.hotrod.tcp_no_delay' can be set in the hibernate.properties file
+ * using 'hibernate.ogm.infinispan_remote.client.infinispan.client.hotrod.tcp_no_delay'.
```

Comment:
I was expecting a short-hand form for the Infinispan properties.

So rather than "hibernate.ogm.infinispan_remote.client.infinispan.client.hotrod.tcp_no_delay" I'd prefer one of:

- _hibernate.ogm.infinispan_remote.client.tcp_no_delay_
- _hibernate.ogm.infinispan_remote.hotrod.tcp_no_delay_